### DISCUSSION TITLE
Widgets for Zebra

### DISCUSSION THREAD

**Discussion by gallantcrusader** [wstyres/Zebra#1646] [Ideas]
I posted this on Reddit, but I was thinking of a widget that either shows the latest news from r/jailbreak or it shows packages that need updates, or even a list of recently updated packages.

**Comment by wstyres** [comment]
We want to try and integrate widgets in the future, but we're focusing on a larger redesign for the app itself at the moment so a widget isn't the top of our priorities. We'll put it on the list for the future!

**Comment by worldlydev** [comment]
> I posted this on Reddit, but I was thinking of a widget that either shows the latest news from r/jailbreak or it shows packages that need updates, or even a list of recently updated packages.

Would be awesome. Can't really think of much else I would want to see in a widget as they aren't interactable by touch with buttons, etc. But both available updates and r/Jailbreak news sounds great.
